src/csharp/Grpc.Core/Internal/NativeLogRedirector.cs

```diff
@@ -86,4 +87,15 @@ private static void HandleWrite(IntPtr fileStringPtr, int line, ulong threadId,
             }
         }
     }
+
+    [AttributeUsage(AttributeTargets.Method)]
+    internal sealed class MonoPInvokeCallbackAttribute : Attribute
```

Comment:
This is here as there only appears to be a single instance of reverse callbacks, but this can go anywhere.

Reply:
This is probably worth a comment that explains why this is defined and that just naming a custom attribute "MonoPInvokeCallback" is enough for the AOT compiler to pick it up (= it doesn't have to a specific attribute class defined in Mono's assemblies).

Reply:
Actually, I see you documented everything in the commit message. Thanks!